src/main/java/com/ibm/crypto/plus/provider/EdDSAPrivateKeyImpl.java

```diff
@@ -131,10 +131,10 @@ public EdDSAPrivateKeyImpl(OpenJCEPlusProvider provider, byte[] encoded)
 
     void checkLength(CURVE curve) throws InvalidKeyException {
 
-        if (CurveUtil.getPrivateCurveSize(curve) != this.h.get().length) {
+        if (CurveUtil.getDEREncodingSize(curve) != this.h.get().length) {
```

Comment:
Not sure this is right here, because the private size map for `EdDSA` had the actual size of the curve and not the `DER` encoding size.

Reply:
I updated this method to use getCurveSize instead of the DER encoding sizes.